plugins/woocommerce/tests/api-core-tests/tests/orders/orders-crud.test.js

```diff
@@ -269,10 +254,19 @@ test.describe( 'Orders API tests: CRUD', () => {
 			);
 		} );
 
+		const delay = ( delayInms ) => {
+			return new Promise( ( resolve ) =>
+				setTimeout( resolve, delayInms )
+			);
+		};
+
 		for ( const expectedOrderStatus of statusesDataTable ) {
 			test( `can update status of an order to ${ expectedOrderStatus }`, async ( {
 				request,
 			} ) => {
+				if ( shouldSkip ) {
+					await delay( 1000 ); // if this runs too fast on an external host, it fails
```

Comment:
This is a bit of a weird one.  If we slow these tests down with a 1 second delay, everything works.  Otherwise they start failing after the second order status.  Maybe a request throttling issue?